manifests/site.pp

```diff
@@ -2,6 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+# NOTE(aerickson): please don't add more global variables.
+#   variables like this should be set in profile modules.
+#   these make testing difficult (this block must be
+#   copy/pasted into every puppet-kitchen manifests).
```

Comment:
What if we remove these and put it into a profile?

Reply:
We could, but then we'd have to set them in each profile... I see the power that the globals afford us here.